DevSkim-DotNet/Microsoft.DevSkim.CLI/Commands/AnalyzeCommand.cs

```diff
@@ -65,7 +65,7 @@ public int Run()
 
             IEnumerable<FileEntry> fileListing;
             Extractor extractor = new Extractor();
-            ExtractorOptions extractorOpts = new ExtractorOptions() { ExtractSelfOnFail = false, DenyFilters = _opts.Globs };
+            ExtractorOptions extractorOpts = new ExtractorOptions() { ExtractSelfOnFail = false, AllowFilters = _opts.IncludeGlobs, DenyFilters = _opts.Globs };
```

Comment:
Do you want to rename _opts.Globs to _opts.DenyGlobs?

Reply:
That makes sense. I held off on doing that because it would be a SemVer change, but these options objects only exist in the CLI so I think its fine actually.